client/cypress/integration/add-user.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { User } from 'src/app/users/user';
+import { AddUserPage } from '../support/add-user.po';
+
+describe('Add user', () => {
+  const page = new AddUserPage();
+
+  beforeEach(() => {
+    page.navigateTo();
+  });
+
+  after(() => {
+    // Lets put the db back to its original state before we leave.
+    //cy.task('seed:database');
```

Comment:
I wasn't sure if we wanted that or not. One one hand, Cypress recommends against resettings things in "after" steps because it makes it harder to inspect things when something goes wrong but at the same time I think it's a good idea to make sure the tests end with the db seeded back to normal since its the same one students use for development.

Reply:
Yeah, I suppose the underlying problem is that it would be preferable to use the `test` database instead of the `dev` database for this. (But we would need a way to communicate that to the server somehow. 🙁)

My first preference would be to remove the `after(() => {})` block entirely, but I'd also be fine with un-commenting the call to `cy.task('seed:database')`. Do you have a preference? 🙂